src/olympia/editors/helpers.py

```diff
@@ -508,13 +508,17 @@ def get_context_data(self):
                 addon = Addon.unfiltered.get(pk=self.addon.pk)
         else:
             addon = self.addon
+        review_url_kw = (
+            {'channel': 'unlisted'} if self.version and
+            self.version.channel == amo.RELEASE_CHANNEL_UNLISTED else {})
+        review_url_kw['addon_id'] = self.addon.pk
```

Comment:
https://github.com/mozilla/addons-server/pull/5047/commits/c9c9ca760eb254bb6f98020eb35d7c8fab1a6645 with tests this time